### DISCUSSION TITLE
Script Editor: alt+number to do symbols

### DISCUSSION THREAD

**Discussion by CritickalGames** [godotengine/godot-proposals#10333] [Engine Core]
I usually use alt+62 to do > or alt+64 to do @ (becouse some key don't work) and i found this little "problem".

I would do the implementation myself, but I have never touched the godot core so I think the task is a bit big for me (I still have to read the page on how to make contributions)

**Comment by AThousandShips** [comment]
Thank you for your proposal, consolidating in:
* https://github.com/godotengine/godot/issues/83294

**Reply by CritickalGames** [reply]
Thanks to you.

Sorry form my english and rooky post. I realy speak spanish and i don't know how do this kind of stuff right. My first time. Sorry.
